src/main/java/ru/evotor/framework/receipt/Position.java

```diff
@@ -650,7 +651,7 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeSerializable(this.priceWithDiscountPosition);
         dest.writeSerializable(this.quantity);
         dest.writeString(this.barcode);
-        dest.writeString(this.mark);
+        writeMark(dest, this.mark);
```

Comment:
dest.writeString(this.mark.rawmark)

...


writeSerializable(this.mark)

Reply:
+